crates/uv/src/commands/tool/list.rs

```diff
@@ -16,6 +16,7 @@ pub(crate) async fn list(preview: PreviewMode, printer: Printer) -> Result<ExitS
     }
 
     let installed_tools = InstalledTools::from_settings()?;
+    let _lock = installed_tools.acquire_lock()?;
```

Comment:
@charliermarsh e.g. here we do not call `init()?` because this is an inspection-only command.